### ISSUE TITLE
Two time series with both Y axes using identical min-max range

### ISSUE THREAD

**Issue by udhos** [wcharczuk/go-chart#83]
I'm plotting two time series into a single graph.
The YAxis range for the first series is distintict than the YAxis range for the secondary series.
I want to plot both series forcing the same YAxis range.
That is, the lowest value for the YAxis should be the lowest y value for both series, and the highest value for YAxis should be taken from the highest y value for both series.
Can this be accomplished easily?

**Comment by james-antill** [comment]
Yes, if I understand you correctly, look at the twoaxis example code ... and pass a Range{Min: x, Max: y} to both y axis (with the same values).
